src/app/address/controllers/address.test.js

```diff
@@ -42,7 +42,7 @@ describe("address controller", () => {
       const generatedAddress = addressFactory(1);
       req.sessionModel.set("addressPostcode", generatedAddress[0].postalCode);
 
-      req.sessionModel.set("chosenAddress", generatedAddress[0]);
```

Comment:
standardised the naming, some places we where using chosenAddress and some places, address.

Stuck with just address. (dont ask me why I used chosenAddress in the first place!)